.github/workflows/main.yml

```diff
@@ -98,33 +98,34 @@ jobs:
           environment-file: environment-dev.yml
           environment-name: xeus
 
-      - name: install cxx compiler
-        shell: cmd /C call {0}
+      - name: micromamba shell hook
+        shell: powershell
         run: |
-          micromamba activate xeus
-          micromamba install cxx-compiler -c conda-forge -y
+          micromamba shell hook -s cmd.exe -p C:\Users\runneradmin\micromamba-root
+
+      - name: Make build directory
+        run: mkdir build
 
       - name: cmake configure
-        shell: cmd /C call {0}
```

Comment:
This line was the cause. It seems that only the first line (usually micromamba activate) was running during all steps.